### ISSUE TITLE
All browser extensions don't work.

### ISSUE THREAD

**Issue by wcimediastudios** [electron-react-boilerplate/electron-react-boilerplate#2788]
(node:14160) ExtensionLoadWarning: Warnings loading extension at C:\Users\User\AppData\Roaming\Electron\extensions\fmkadmapgofadopljbjfkapdkoienihi: Unrecognized manifest key 'browser_action'. Unrecognized manifest key 'minimum_chrome_version'. Unrecognized manifest key 'update_url'. Cannot load extension with file or directory name _metadata. Filenames starting with "_" are reserved for use by the system.
10:01:28.657 > Skip checkForUpdatesAndNotify because application is not packed

**Comment by timsgardner** [comment]
Same issue (with same fmkad.... id).

**Comment by timsgardner** [comment]
I found an extremely unpleasant solution. Following the suggestion [here](https://github.com/MarshallOfSound/electron-devtools-installer/issues/164#issuecomment-810296252):
- Add the latest (unreleased) build of [electron-devtools-installer](https://github.com/MarshallOfSound/electron-devtools-installer) via `yarn add git+https://github.com/MarshallOfSound/electron-devtools-installer.git`, as suggested [here](https://github.com/MarshallOfSound/electron-devtools-installer/issues/176#issuecomment-787513395).
-  Go into the directory of the extensions themselves. For React Developer Tools on my (Windows) machine, that's `C:\Users\<user>\AppData\Roaming\Electron\extensions\fmkadmapgofadopljbjfkapdkoienihi`.
   - Delete the `_metadata` directory.
   - In `manifest.json`, delete the `update_url`, `minimum_chrome_version`, and `browser_action` entries.
- Upgrade Electron to `12.0.2` (the latest as of writing) via `yarn upgrade electron@12.0.2`.
- In `main.dev.ts`, when creating the new `BrowserWindow`, set `contextIsolation` to `false` (I can't vouch for security implications etc of this):
```Typescript
  mainWindow = new BrowserWindow({
    show: false,
    width: 1024,
    height: 728,
    icon: getAssetPath('icon.png'),
    webPreferences: {
      nodeIntegration: true,
      contextIsolation: false,
    },
  });
```
Then it should work. Clearly not a great solution but perhaps a stopgap. The problem seems to be spread across multiple infrastructure repos, I'm not sure if extensions are working for anyone right now. Then again if they were broken for everyone I'd expect more noise on this issue, I had to dig to find this workaround.
